<launch>
  <!-- Parameters from the file -->
  <rosparam file="$(find large_scale_drawing)/data/config/half_sphere_grid.config" />
  <param name="/iiwa/toolName" type="string" value="tool" />

<!--    --><!-- iiwa line -->
<!--   <node name="drawing_manager" pkg="large_scale_drawing" type="drawing_manager" respawn="false" output="screen" /> -->

</launch>
